src/ReCaptcha.php

```diff
@@ -368,85 +354,93 @@ public function setPrivateKey($privateKey)
      *
      * This method uses the public key to fetch a recaptcha form.
      *
-     * @param null|string $name Base name for recaptcha form elements
+     * @param null|string $name
+     *            Base name for recaptcha form elements
      * @return string
-     * @throws Exception
+     * @throws \ZendService\ReCaptcha\Exception
      */
-    public function getHtml($name = null)
+    public function getHtml()
     {
-        if ($this->publicKey === null) {
-            throw new Exception('Missing public key');
+        if ($this->siteKey === null) {
+            throw new Exception('Missing site key');
         }
 
         $host = self::API_SERVER;
 
-        if ((bool) $this->params['ssl'] === true) {
-            $host = self::API_SECURE_SERVER;
-        }
-
-        $htmlBreak = '<br>';
-        $htmlInputClosing = '>';
-
-        if ((bool) $this->params['xhtml'] === true) {
-            $htmlBreak = '<br />';
-            $htmlInputClosing = '/>';
+        // Should We use a onload callback?
+        if (!empty($this->options['onload'])) {
```

Comment:
Space following `!` please (for internal consistency; you've done that everywhere else I've seen! :smile:)